vtm.js

```diff
@@ -190,28 +252,103 @@ const app = Vue.createApp({
           },
         ],
       },
+      disciplines:{
+        id: "Disciplines",
+        resource: [0,1,1,0,0,0],
+        data: [
+          {
+            id: 'Animalism',
+            value: 0,
+          },
+          {
+            id: 'Auspex',
+            value: 0,
+          },
+          {
+            id: 'Blood Sorcery',
+            value: 0,
+          },
+          {
+            id: 'Celerity',
+            value: 0,
+          },
+          {
+            id: 'Dominate',
+            value: 0,
+          },
+          {
+            id: 'Fortitude',
+            value: 0,
+          },
+          {
+            id: 'Obfuscate',
+            value: 0,
+          },
+          {
+            id: 'Potence',
+            value: 0,
+          },
+          {
+            id: 'Presence',
+            value: 0,
+          },
+          {
+            id: 'Protean',
+            value: 0,
+          },
+        ],
+      },
     };
   },
+  
   template: `
-  <stat-section 
+  <character-info
+    :bio="biography"
+    :clans="clans">
+  </character-info>
+  <attribute-section 
     :stats="attributes"
-    @stat-section-change="$event[0].value=$event[1]"
-    >
-  </stat-section>
-  <stat-section 
+    @stat-section-change="$event[0].value=$event[1]">
+  </attribute-section>
+  <skill-section 
     :stats="skills"
-    @stat-section-change="$event[0].value=$event[1]"
-    >
-  </stat-section>`
+    :distributions="skillDistributions"
+    @stat-section-change="$event[0].value=$event[1]">
+  </skill-section>
+  <discipline-section 
+    v-if="biography.clan"
```

Comment:
imho ak neni clan definovany, tak je Caitiff. Takze by to mohlo byt default

Reply:
hej lenze caitiff aj thin-blood maju `clan.abilities`nastavene na `null` takze to mozes rozlisit len podla blood potency
ja by som daval default klan co ma null, len problem je blood potency a neviem jak to pojde... proste to pridam ako issue

Reply:
takze nejde urobit podmienku:

if clan == Thin Blood ?

Reply:
no tak asi ide... ale ked to ma byt customizable, tak by malo co najviac veci ist nastavit cez data, clovek by nemal byt nuteny rozumiet js a menit ho

Reply:
imho ked budes chciet kazdu vynimku nacpat do dat, tak ti vznikne slozita struktura a pridat jednu malu podmienku niekde do kodu je dobry kompromis.

bohuzial nic neni idealne a neda sa to robit absolutne, len matematika :-)

Reply:
Teraz ked na to pozeram, tak vidim, ze teraz niesu data postavy oddelene od obmedzeni pravidiel. Ale neviem ci ma zmysel sa na to sustredit